packages/web-app-files/src/composables/actions/spaces/useSpaceActionsUploadImage.ts

```diff
@@ -68,49 +60,65 @@ export const useSpaceActionsUploadImage = ({
       })
     }
 
-    return loadingService.addTask(() => {
-      return clientService.webdav
-        .putFileContents(selectedSpace, {
+    const extraHeaders = {
+      'content-type': 'application/offset+octet-stream'
+    }
```

Comment:
Now you have `extraHeaders` and `headers` 😄

Reply:
shiddo, thx for pointer